Classes/Compiler/AbstractMenuCompiler.php

```diff
@@ -78,4 +78,21 @@ protected function getCurrentSite(): ?SiteInterface
     {
         return $GLOBALS['TYPO3_REQUEST']->getAttribute('site');
     }
+
+    /**
+     * Function to parse typoscript config with stdWrap
+     * @param string $content
+     * @param string $configuration
+     *
+     * @return string
+     */
+    public function parseStdWrap($content, $configuration): string
+    {
+        $return = GeneralUtility::makeInstance(ContentObjectRenderer::class)->stdWrap($content, $configuration);
```

Comment:
one should try to use the main cObject object as much as possible (which we are handing in the main class anyway). instantiating a new cObject can lead to side effects with stdWrap. e.g. `data` is not set (this would require `$cObj->start()`)